proto/import_sstpb.proto

```diff
@@ -284,10 +284,30 @@ message DuplicateDetectResponse {
     repeated KvPair pairs = 3;
 }
 
-message ApplyRequest {
+message KVMeta {
     // The file name of the KV file.
     string name = 1;
 
+    // file length for check.
+    uint64 length = 2;
+
+    // tell us which cf should apply. WRITE_CF or DEFAULT_CF e.g.
+    string cf = 3;
+
+    // is_delete represents whether we should delete the kv in tikv.
+    // it may not be too much delete file. only rollBack operation will generate delete kv file.
+    bool is_delete = 4;
+
+    // the key ts space large than restore_ts will filter.
+    // because key is sorted by ts already.
+    uint64 restore_ts = 5;
+}
+
+
+message ApplyRequest {
+    // The meta of the KV file.
+    KVMeta meta = 1;
```

Comment:
🤔 use a different number?

Reply:
🤔 maybe it's okay since only `br-stream` use `ApplyReqeust` for now.

Reply:
🤔 okay